playbooks/development.yml

```diff
@@ -1,37 +0,0 @@
----
```

Comment:
This one is used by `scripts/provisioning-dev.sh` but I'm not sure anyone is using it but it'll come in handy when we start changing monitoring and logging settings for Postgres for instance.

Reply:
Ah I see, it's used with the LXC dev setup. Do we know who actually uses it?

Reply:
I know @enricostano and @danypr92 used to but they don't do any OFN since ages. I just wonder if I could make use of it while playing with postgres.conf soon.

Reply:
The whole `/scripts` folder is just for setting up LXC containers as dev environments. If nobody uses it, we can remove the folder.

Reply:
Ah, ok. Well lets leave it then.

Reply:
yep, let me give it a try and see if it still works.

Reply:
I've reverted the deletion of `development.yml`. :+1:

Reply:
If I remember correctly, that was a community contribution a few years ago. I don't think any core devs ever used it.